[
  {
    "cmd": [],
    "name": "RECIPE CRASH (Uncaught exception)",
    "~followup_annotations": [
      "@@@STEP_EXCEPTION@@@",
      "The recipe has crashed at point 'Uncaught exception'!",
      "",
      "Traceback (most recent call last):",
      "  File \"RECIPE_REPO[flutter]/recipe_modules/web_util/examples/full.py\", line 19, in RunSteps",
      "    api.web_util.prepare_web_dependencies(engine_checkout_path)",
      "  File \"RECIPE_REPO[flutter]/recipe_modules/web_util/api.py\", line 86, in prepare_web_dependencies",
      "    raise ValueError('Web Dependency %s not available.' % dep)",
      "ValueError('Web Dependency invalid_dependency not available.')"
    ]
  },
  {
    "failure": {
      "humanReason": "Uncaught Exception: ValueError('Web Dependency invalid_dependency not available.')"
    },
    "name": "$result"
  }
]